engine/lib/phx/src/audio/sound.rs

```diff
@@ -1,32 +1,25 @@
 use std::cell::RefCell;
 use std::rc::Rc;
-use std::time::Duration;
 
 use internal::ConvertIntoString;
 
 use crate::math::*;
 
-use kira::sound::static_sound::{StaticSoundData, StaticSoundHandle, StaticSoundSettings};
-use kira::sound::PlaybackState;
+use kira::sound::static_sound::{StaticSoundData, StaticSoundSettings};
 use kira::sound::{EndPosition, PlaybackPosition, Region};
 use kira::spatial::emitter::EmitterHandle;
-use kira::tween::{Easing, Tween};
-use kira::StartTime;
+use kira::tween::Tween;
 
-use super::process_command_error;
+use super::{process_command_error, SoundInstance};
 
 pub struct Sound {
     path: String,
     sound_data: StaticSoundData,
     emitter: Option<EmitterHandle>,
-    sound_handle: Option<Rc<RefCell<StaticSoundHandle>>>,
+    pub instances: Vec<Rc<RefCell<SoundInstance>>>,
```

Comment:
Better to keep accessibility of all fields the same. Make it private and add getter method.